packages/odyssey/CHANGELOG.md

```diff
@@ -5,7 +5,43 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.6.0] - 2020-10-01
+## [0.7.0] - 2020-11-18
+
+### Added
```

Comment:
In general, can we separate out `[docs]` as it does not affect any downstream consumers?

Reply:
We've had requests on both sides of this. In the future, I'm fine separating things out, but then we'll need to separate their versioning as well. As it stands, Odyssey's Workspace has one changelog and one version.

Reply:
I am fine with one versioning/changelog, but can we separate out docs into a separate section, so that actual consumers don't have to worry about that part? makes for easier reading and lesser confusion. As it stand, for anyone unaware, they won't know what `adds "prism-theme-odyssey" package for styling code blocks` or other doc items actually mean and what impact it has on downstream consumers.

Reply:
Is this a blocking change? I'm up for reformatting our CHANGELOG, but we'll also need to back-update the release notes in GH to have parity. I'd rather not rush that as we're trying to get this out.

Reply:
As a compromise, can we move all the `[docs]` items as the last ones within each section - added, removed, changed? Rather than having them in between component changes.

Reply:
Updated!

Reply:
I've also opened a ticket, so we don't forget to address this as we rethink our packages: https://github.com/okta/odyssey/issues/835

Reply:
+1. It's confusing that we're coupling all CHANGELOG entries within `packages/odyssey`. Instead, each package should have their own CHANGELOG.